OpenRA.Game/Traits/Target.cs

```diff
@@ -173,6 +163,48 @@ public bool IsInRange(WPos origin, WDist range)
 			return Positions.Any(t => (t - origin).HorizontalLengthSquared <= range.LengthSquared);
 		}
 
+		public bool IsOwnerTargetable(Actor self)
```

Comment:
`self` is used in traits where it implicitly refers to the owning actor.  That isn't the case here, so please rename this to `targeter` to match `IsValidFor`.